docs/contents/news/1-19/1/CHANGELOG-1.19.1.md

```diff
@@ -0,0 +1,53 @@
+# Changelog for v.1.19.1
+
+This changelog highlights the changes from EKS-D [v.1.18.1](https://github.com/aws/eks-distro/tree/v1-18-eks-1) to 
+[v.1.19.1](https://github.com/aws/eks-distro/tree/v1-19-eks-1).
+
+## Upgrade Kubernetes Versions
+
+* Upgraded Kubernetes to 1.19
+* Upgraded dependencies and components
+
+## Patches 
+
+### Patches Removed
+_The following patches were in EKS-D [v1.18](https://github.com/aws/eks-distro/tree/v1-18-eks-1/projects/kubernetes/kubernetes/1-18/patches)
+but were removed in the version._
+
+**Patches that were removed because their changes made in upstream Kubernetes 1.19:**
+* 0007-EKS-PATCH-fix-aws-loadbalancer-nodePort-cannot-chang.patch
+  * See [PR #89562](https://github.com/kubernetes/kubernetes/pull/89562)
+* 0008-EKS-PATCH-aws-Fix-address-sorting-of-multiple-interf.patch
+  * See [PR #91889](https://github.com/kubernetes/kubernetes/pull/91889)
+* 0009-EKS-PATCH-fix-aws-loadbalancer-vpc-cidr-calculation.patch
+  * See [PR #92227](https://github.com/kubernetes/kubernetes/pull/92227)
+* 0010-EKS-PATCH-refine-aws-loadbalancer-worker-node-SG-rul.patch
+  * See [PR #92224](https://github.com/kubernetes/kubernetes/pull/92224)
+* 0011-EKS-PATCH-Allow-UDP-for-AWS-NLB.patch
+  * See [PR #92109](https://github.com/kubernetes/kubernetes/pull/92109)
+
+**Patches that were removed for another reason:**
+* **0013-EKS-PATCH-aws_credentials-update-ecr-url-validation-.patch**
+  * Adds support for ECR endpoints in isolated AWS regions. There is an open PR ([#95415](https://github.com/kubernetes/kubernetes/pull/95415))
+    for upstream Kubernetes that applies these changes.
+  * This patch will likely be added back to a future minor release for 1.19.
+
+### Patches Added
+_The following patches were mot in EKS-D [v1.18](https://github.com/aws/eks-distro/tree/v1-18-eks-1/projects/kubernetes/kubernetes/1-18/patches)
+but were removed in the version._
+
+* **0010-EKS-PATCH-additional-subnet-configuration-for-AWS-el.patch** 
+  * This patch applies changes from upstream Kubernetes [PR #97431](https://github.com/kubernetes/kubernetes/pull/97431).  
+    At of the time of this changelog's creation, this PR is tagged to be part of the Kubernetes v1.21 release.
```

Comment:
At o̶f̶ the

Reply:
Yeah I couldn't decide between "As of the time of..." and "At the time of...", but apparently somehow combined them both in a way that makes no sense. I'll fix this